github-connector/internal/httperrors/httperrors.go

```diff
@@ -0,0 +1,69 @@
+package httperrors
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/kyma-incubator/hack-showcase/github-connector/internal/apperrors"
+	log "github.com/sirupsen/logrus"
+)
+
+type ErrorResponse struct {
+	Code  int    `json:"code"`
+	Error string `json:"error"`
+}
+
+func AppErrorToResponse(appError apperrors.AppError, detailedErrorResponse bool) (status int, body ErrorResponse) {
+	httpCode := errorCodeToHttpStatus(appError.Code())
+	errorMessage := appError.Error()
+	return formatErrorResponse(httpCode, errorMessage, detailedErrorResponse)
+}
+
+func errorCodeToHttpStatus(code int) int {
+	switch code {
+	case apperrors.CodeInternal:
+		return http.StatusInternalServerError
+	case apperrors.CodeNotFound:
+		return http.StatusNotFound
+	case apperrors.CodeAlreadyExists:
+		return http.StatusConflict
+	case apperrors.CodeWrongInput:
+		return http.StatusBadRequest
+	case apperrors.CodeUpstreamServerCallFailed:
+		return http.StatusBadGateway
+	case apperrors.CodeAuthenticationFailed:
+		return http.StatusForbidden
```

Comment:
In this case we should use 401 instead of 403, you can check it [right here](https://stackoverflow.com/questions/50143518/401-unauthorized-vs-403-forbidden-which-is-the-right-status-code-for-when-the-u?rq=1)

``` suggestion
return http.StatusUnauthorized
```